README.md

```diff
@@ -1,6 +1,100 @@
-# roku-dev-cli
+# Overview
 
-## Usage
+*roku-dev-cli* is a command-line tool for loading builds onto a dev-enabled Roku device.  
 
-`roku --help`
+Features:
+* build zipping, installation, and logging
+* project analysis
+* network proxying
 
+# Installation
+
+*roku-dev-cli* can be installed from the pypi repository using pip.  
+
+```shell
+pip install roku-dev-cli
+```
+
+# Loading a Build
+
+To load a build on the Roku, first make sure the device is on the same network and is in developer mode ([how to enter developer mode](https://sdkdocs.roku.com/display/sdkdoc/Loading+and+Running+Your+Application#LoadingandRunningYourApplication-EnablingDevelopmentModeonyourbox)).  
+
+In your terminal, go to the base directory of your Roku project where the *manifest* file exists.  Then run the following command:
+
+```shell
+roku <IP_ADDRESS>
+```
+This will zip up the current directory and send it to the device.  If the app is successfully installed the tool will output the application's log to your console as it executes.  
+
+The script will do some basic checking of the project making sure all the component files are valid XML.  
+
+# Loading a build from an existing zip file
+If you already have a zip file you want to load, you can use the zip_file flag instead of zipping the current directory:
+
+```shell
+roku --zip_file <ZIP_FILE_PATH> <IP_ADDRESS>
+```
+
+This will install the zip file and start logging the application output.  
```

Comment:
Consider mentioning that – unlike the `telnet` interface, pressing Control-C here is the correct way to stop logging things from the device and won’t actually stop execution 🙂